#include "CustomException.h"

using namespace Mistake;

CustomException::CustomException(std::string msg):message(msg) {}
